### ISSUE TITLE
Show waiting assignments

### ISSUE THREAD

**Issue by Aech27** [logicguy1/FOSS-Lectio-improvements#8]
A button like "Skjul afleveret opgaver", that shows returned assignments.

**Comment by logicguy1** [comment]
Maybe it would be better to add it as the default behavior? Maybe we should change it so that the function that creates the buttons takes a function that gets called on every row and then returns true or false to wheater or not to hide that row, would allow for more complex functionalaty VS the current setup with only text filters being supported

**Comment by logicguy1** [comment]
Or maybe add a filter for the row called Afventer

**Comment by logicguy1** [comment]
Added in 34e2406 however there is a small issue with adding multiple filters and overwriting old filters

**Comment by Reikimann** [comment]
> Added in [34e2406](https://github.com/logicguy1/FOSS-Lectio-improvements/commit/34e2406deede5190c5d0b0e16b9e4d649f71e5b5) however there is a small issue with adding multiple filters and overwriting old filters

Yeah. Toggling hideWaiting button, when hideDelivered is on. Delivered assignments is shown. 
I see two possibilities: 

1. Maybe decoupling the hideWaiting function from createAssingmentField.
2. Add a new param to the createAssingmentField to ignore certain toggles when other toggles is on.

**Comment by Reikimann** [comment]
![image](https://github.com/logicguy1/FOSS-Lectio-improvements/assets/78156411/7490b763-9ebf-4c91-b998-5d570668cdb3)

After loading the site, delivered assignments are shown even if they are toggled of (as can be seen on the picture).

@logicguy1 I suggest that we remove this toggle for now. This feature shouldn't be shipped to the users until it works properly. See PR #23
